src/elm/Broxy.elm

```diff
@@ -29,6 +32,7 @@ main =
 type alias Model =
     { proxy : Maybe ISocksProxy
     , error : Maybe String
```

Comment:
as there can be multiple subscriptions, singular error value is not sufficient. 
I'm thinking on how encapsulate ISocksProxy and its error into type

Reply:
One way would be to a separate sum type for errors:
```elm
type Error =
    ProxyStartupError String
  | AdblockerStartupError String
  ...
```

Then the `Model` could become:
```elm
type alias Model =
    { proxy : Maybe ISocksProxy
    , error : Maybe String
    }
```

Although, now that I think about it... we should probably decide what should be in the `Model`. Does it even make sense to have an `error` field? It all depends on what we want to display in the UI I guess?

Reply:
have no opinion on that yet. 

but atm the UI is reacting to messages from the background so technically there is now way to get an error, right?

Reply:
No exception should come from the Elm side, so we would only reflect exceptions from Cliqz in the view.